scripts/test_snapshots_v2.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
I would still prefer this as a system test, as this never run anywhere. Can we maybe remove this and create an issue for it?